mh/pat.py

```diff
@@ -2438,9 +2466,24 @@ def sendAnsLayerUp(self, data, seq):
         JP: レイヤアップ返答
         TR: Layer up response
         """
-        self.session.layer_up()
+        self.notify_layer_departure(False)
         self.send_packet(PatID4.AnsLayerUp, b"", seq)
 
+    @staticmethod
```

Comment:
Unless I'm mistaken, I don't think that `@staticmethod` is necessary, especially if not called from outside this class.

Reply:
As for this, I was getting this from PyLance (Withot `@staticmethod`)

![image](https://github.com/sepalani/MH3SP/assets/2200611/3a5b75c3-ff4e-4330-9bab-0163d12c0d10)

Reply:
Odd. Maybe the `self` parameter was missing or the type hints comment needs to be adjusted?